senza/configuration.py

```diff
@@ -0,0 +1,76 @@
+from collections.abc import MutableMapping
+from pathlib import Path
+from typing import Dict, Tuple
+
+import yaml
+from click import get_app_dir
+
+from .exceptions import InvalidConfigKey
+
+
+class Configuration(MutableMapping):
```

Comment:
this whole class looks complicated for what it does: load/store a dict as YAML

Also we already use the very naive approach with `stups_cli.load_config` in all the STUPS CLI tools.